Task6.2/Task6.2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task6._2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+           Console.WriteLine("Введите слово чемпионат");
+           string h = Console.ReadLine();
```

Comment:
Не вижу смысла заставлять пользователя вводить слово с клавиатуры.
```suggestion
           Console.WriteLine("Начальное слово \"чемпионат\"");
           string h = " чемпионат";
```